assign_rights/forms.py

```diff
@@ -28,58 +30,45 @@ class Meta:
+            Div(
+                Div(Field("rights_end", template="forms/custom_field.html", v_model="rightsEnd")),
+                Div(Field("end_date", pattern=r"\d{4}-\d{2}-\d{2}", required="required"), v_if="rightsEnd=='end_date'"),
+                Div(Field("end_date_period", required="required"), v_if="rightsEnd=='end_date_period'"),
+                Div(Hidden(name="end_date_open", value="true",), v_if="rightsEnd=='end_date_open'")),
+            Div("basis_note")
+        )
 
+
+class RightsShellForm(ModelForm):
     rights_begin = ChoiceField(
         label="Start of Rights",
         choices=(("", "---------"),
                  ("start_date", "These rights start on a specific date"),
                  ("start_date_period", "These rights start after an embargo period"),
-                 ("start_date_period_zero", "These rights start on creation date of materials")),
-        widget=Select(attrs={'required': True, 'v-model': 'rightsBegin'}))
+                 ("start_date_period_zero", "These rights start on creation date of materials")))
     rights_end = ChoiceField(
         label="End of Rights",
         choices=(("", "---------"),
                  ("end_date", "These rights end on a specific date"),
                  ("end_date_period", "These rights end after an embargo period"),
-                 ("end_date_open", "There is no end date for these rights")),
-        widget=Select(attrs={'required': True, 'v-model': 'rightsEnd'}))
-
-    class Meta:
-        model = RightsShell
-        fields = [
-            'rights_begin',
-            'rights_end',
-            'start_date',
-            'end_date',
-            'start_date_period',
-            'end_date_period',
-            'end_date_open'
-        ]
-        labels = {
-            'start_date_period': "Start Date Embargo Period (in years)",
-            'end_date_period': "End Date Embargo Period (in years)",
-            'start_date': "Start Date (yyyy-mm-dd)",
-            'end_date': "End Date (yyyy-mm-dd)",
-        }
-        widgets = {
-            'start_date': TextInput(attrs={'required': True, 'pattern': "\\d{4}-\\d{2}-\\d{2}"}),
-            'start_date_period': TextInput(attrs={'required': True}),
-            'end_date': TextInput(attrs={'required': True, 'pattern': "\\d{4}-\\d{2}-\\d{2}"}),
-            'end_date_period': TextInput(attrs={'required': True}),
-            'end_date_open': HiddenInput(attrs={'value': True, 'type': 'hidden'}),
-        }
-
-
-class RightsShellForm(ModelForm):
-    dates = RightsShellDates()
+                 ("end_date_open", "There is no end date for these rights")))
 
     def __init__(self, *args, **kwargs):
-        kwargs.setdefault('label_suffix', '')
         super().__init__(*args, **kwargs)
+        self.helper = FormHelper(self)
+        self.helper.field_template = "forms/custom_field.html"
+        self.helper.form_tag = False
+        self.helper.disable_csrf = True
```

Comment:
This seems...unexpected. Any insight into why this is disabled?

Reply:
I can't recall a reason - I guess just make sure the form is still functional after you remove it?